### ISSUE TITLE
1NT intervention

### ISSUE THREAD

**Issue by Muppet1** [EdwardPiwowar/BBA#401]
![image](https://github.com/EdwardPiwowar/BBA/assets/11272970/77f1afcc-c9eb-42a4-bfa3-be2305f3e705)
0CBF616DF850F083C4981335EC12

table one.  West correctly doubles.  East now has a choice of sensible bids 3h 4h 3NT.   It of course chooses a 3 card suit at the 4 level in a minor instead.

**Comment by EdwardPiwowar** [comment]
Next BBA
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/9fa3b08c-0b9a-4b1c-b938-4ff566612520)

**Comment by ThorvaldAagaard** [comment]
4D is to optimistic, just bid 4H

**Comment by EdwardPiwowar** [comment]
4D may be needed if opponents bid again. Then 4D makes the decision easier.

**Comment by ThorvaldAagaard** [comment]
Yes, point taken, but when bidding 4D both opponents have passed, so why should they bid again?

Over 3H, 4D is slammish, as we are committed to game, it should be shortage.
Here West is protected by initial pass.

If you want to keep the support cuebid, the explanation should be changed to: To strong for just bidding 4H
